### PR TITLE
Agent ID for TestCafe is eyes.webdriverio

### PR DESCRIPTION
**Pull request by chaim-aharonson-applitools**
My first commit :-)

### CODE REVIEW

**Comment by kyryloonufriiev** [comment]
Congrats on your first commit :) You have to add a changelog entry to this SDK.

**Comment by chaim-aharonson-applitools** [comment]
> Congrats on your first commit :) You have to add a changelog entry to this SDK.

after it failed I did update the changelog but it still fails, why?

File: packages/eyes-testcafe/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 ## Unreleased
 
 
+## 1.15.3 - 2022/1/19
+
+- updated 'Agent ID' to 'eyes.testcafe' (from 'eyes.webdriverio')
```
Comment (kyryloonufriiev) [inline packages/eyes-testcafe/CHANGELOG.md:8]: You should not create your own heading here, but add an item under the `Unreleased` header. Then when we decide to actually release our automation will create a header with the version.

**Review by kyryloonufriiev — CHANGES_REQUESTED** [review]

**Review by kyryloonufriiev — APPROVED** [review]

**Comment by kyryloonufriiev** [comment]
You can squash and merge it now

**Pull request merged**